teamA/gisuk/app.js

```diff
@@ -28,11 +28,39 @@ appDataSource.initialize()
 app.use(cors());
 app.use(morgan('dev'));
 app.use(express.json());
-
+    
 app.get("/ping", (req,res) => {
   res.status(200).json({"message" : "pong"});
 })
 
+app.get("/users", async (req,res) => {
+  await appDataSource.query(
+    `SELECT 
+    users.name,
+    users.email,
+    users.profile_image
+    FROM users`,
+    (err, rows) => {
+      res.status(200).json(rows);
+    }
+  )
+});
+
+app.post("/users", async (req, res, next) => {
+  const { name, email, profile_image, password} = req.body  
+    await appDataSource.query(
+      `INSERT INTO users(
+        name,
+        email,
+        profile_image,
+        password
+        ) VALUES (?, ?, ?, ?);
+        `,
+        [name, email, profile_image, password]
+      );
+    res.status(201).json({message : "usercreated"});
```

Comment:
```suggestion
    res.status(201).json({message : "userCreated"});
```
다소 마이너한 리뷰이지만 디테일 한 부분들도 신경쓴다면 더욱 코드에 대한 가독성을 높일 수 있습니다.